distribution/lib/Standard/Visualization/0.0.0-dev/src/SQL/Visualization.enso

```diff
@@ -45,5 +45,5 @@ find_expected_enso_type_for_sql sql_type =
     if sql_type.is_definitely_integer then "Standard.Base.Data.Numbers.Integer" else
         if sql_type.is_definitely_double then "Standard.Base.Data.Numbers.Decimal" else
             if sql_type.is_definitely_text then "Standard.Base.Data.Text.Text" else
-                if sql_type.is_definitely_boolean then "Standard.Base.Boolean.Boolean" else
+                if sql_type.is_definitely_boolean then "Standard.Base.Data.Boolean.Boolean" else
                     Nothing
```

Comment:
I realise this whole function should be re-implemented - I'm not sure maybe we didn't have the needed functionality back when this was being done. But now we should implement it as follows:
```
expected_type = if sql_type.is_definitely_integer then Integer else
        if sql_type.is_definitely_double then Decimal else
            if sql_type.is_definitely_text then Text else
                if sql_type.is_definitely_boolean then Boolean else
                   Nothing
case expected_type of
    Nothing -> Nothing
    _ -> Meta.get_qualified_type_name expected_type
```

There is no need now to duplicate the fully qualified type name and bake it into code, if we can easily get it programmatically - and now in case of any future changes, this method will not need to be updated :)

Can I please ask you to add this change? Or I can also push it if you want